src/services/AsyncStorage.ts

```diff
@@ -109,12 +109,14 @@ const compareFavorites = (favorite1: Favorite, favorite2: Favorite) =>
   favorite1.id === favorite2.id && favorite1.documentType === favorite2.documentType
 
 const migrateToNewFavoriteFormat = async (): Promise<void> => {
-  const documents = await AsyncStorage.getItem(FAVORITES_KEY)
-  const parsedDocuments = documents ? JSON.parse(documents) : []
-  if (parsedDocuments.length === 0) {
+  const vocabularyItems = await AsyncStorage.getItem(FAVORITES_KEY)
+  const parsedVocabularyItems = vocabularyItems ? JSON.parse(vocabularyItems) : []
+  if (parsedVocabularyItems.length === 0) {
     return
   }
-  await setFavorites(parsedDocuments.map((item: number) => ({ id: item, documentType: DOCUMENT_TYPES.lunesStandard })))
+  await setFavorites(
+    parsedVocabularyItems.map((item: number) => ({ id: item, documentType: DOCUMENT_TYPES.lunesStandard }))
```

Comment:
This needs to be renamed still

Reply:
what exactly do you mean?

Reply:
i didn't rename it due to it's relation with the api

Reply:
Is not related to the api, it should be safe to rename. It's just an internally used type/id.